templates/frontend/objects/issue_toc.tpl

```diff
@@ -78,6 +78,26 @@
 			{/if}
 		{/foreach}
 
+		{* DOI *}
+		{assign var=doiObject value=$issue->getData('doi')}
+		{if $doiObject}
+			{assign var="doiUrl" value=$doiObject->getData('resolvingUrl')|escape}
+			<div class="pub_id doi">
+				<span class="type">
+					DOI:
+				</span>
+				<span class="id">
+					{if $doiUrl}
+						<a href="{$doiUrl|escape}">
+								{$doiUrl}
+							</a>
+					{else}
+						{$doiObject->getData('doi')|escape}
+					{/if}
```

Comment:
I think the `if` is unnecessary. We'll always get a url if we have a DOI, right?